plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -11,56 +11,66 @@
+            See U(https://github.com/lxc/lxd/blob/lxd-4.6/doc/rest-api.md#put-etag-supported-2)'
+          - If the instance already exists and its "config" entry in metadata
             obtained from
-            GET /1.0/containers/<name>
-            U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersname)
-            are different, they this module tries to apply the configurations.
-          - The key starts with 'volatile.' are ignored for this comparison.
-          - Not all config values are supported to apply the existing container.
-            Maybe you need to delete and recreate a container.
+            GET /1.0/instances/<name>
+            U(https://github.com/lxc/lxd/blob/lxd-4.6/doc/rest-api.md#10instancesname)
+            is different, this module tries to apply the configuration values.
+          - Keys starting with 'volatile.' are ignored for this comparison.
+          - Not all config items can be applied to existing or running instances.
+            You may need to restart or delete and recreate the instance.
         required: false
     devices:
         description:
-          - 'The devices for the container
-            (e.g. { "rootfs": { "path": "/dev/kvm", "type": "unix-char" }).
-            See U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-1)'
+          - 'The devices for the instance
+            (e.g. { "kvmpassthrough": { "path": "/dev/kvm", "type": "unix-char" }).
+            See U(https://github.com/lxc/lxd/blob/lxd-4.6/doc/rest-api.md#post-optional-targetmember)'
         required: false
     ephemeral:
         description:
-          - Whether or not the container is ephemeral (e.g. true or false).
-            See U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-1)
+          - Whether or not the instance is ephemeral (e.g. true or false).
+            See U(https://github.com/lxc/lxd/blob/lxd-4.6/doc/rest-api.md#post-optional-targetmember)
         required: false
         type: bool
     source:
         description:
-          - 'The source for the container
+          - 'The source for the instance
             (e.g. { "type": "image",
                     "mode": "pull",
                     "server": "https://images.linuxcontainers.org",
-                    "protocol": "lxd",
+                    "protocol": "simplestreams",
                     "alias": "ubuntu/xenial/amd64" }).'
-          - 'See U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-1) for complete API documentation.'
+          - 'See U(https://github.com/lxc/lxd/blob/lxd-4.6/doc/rest-api.md#post-optional-targetmember) for complete API documentation.'
           - 'Note that C(protocol) accepts two choices: C(lxd) or C(simplestreams)'
         required: false
+    type:
+        choices:
+          - container
+          - virtual-machine
+        description:
+          - Define the type of instance to be launched
```

Comment:
```suggestion
          - Define the type of instance to be launched.
```
Descriptions should always be complete sentences (i.e. with a period at the end).